.github/workflows/Test.yml

```diff
@@ -11,12 +11,8 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v2
-      - name: List available Xcode versions
-        run: ls /Applications | grep Xcode
-      - name: Force Xcode 12.4
-        run: sudo xcode-select -switch /Applications/Xcode_12.4.app
-      - name: List available simulators
-        run: xcrun simctl list
```

Comment:
Removed these since that information is out in each runner: https://github.com/actions/runner-images/blob/main/images/macos/macos-12-Readme.md